src/trip.c

```diff
@@ -284,3 +285,59 @@ Trips* GetTrips(CURL* curl, const char* origin, const char* destination) {
 
   return trips;
 }
+
+void CoordinatesForStations(CURL* const curl) {
```

Comment:
@TexWO The function should return a pointer to some kind of struct that has a pointer to an array of `Coordinates` structs and a `size_t` that is the number of coordinates, or something like that. Otherwise the data cannot be processed further.